src/views/auth.hbs

```diff
@@ -5,5 +5,13 @@
     <input id="registerName" name="name" type="text" placeholder="Enter a name..."/>
     <input id="registerPassword" name="password" placeholder="Enter a password..." type="password"/>
     <button id="registerSubmit" name="registerButton" type="submit" name="submit">Register</button>
-</form>
-<script src="/js/register.js"></script>
+  </form>
+
+
+  <form id='loginForm'>
+    <input type="text" placeholder="Username" name= 'loginUsername' required id="loginUsername"></br>
```

Comment:
You shouldn't really be using breaks here, I would do this in css. This can be done later.